### PR TITLE
adding gems

### PR DESCRIPTION
**Pull request by cmcfadzean**
Moves Gems so they will be dependencies of the gem. 
No longer requiring we add them to the parent app.

### CODE REVIEW

File: app/views/sage/application/_assistant.html.erb [hunk 1]
```diff
@@ -5,10 +5,6 @@
   </button>
   <%= image_tag("sage/sage.svg", class: "sage-assistant__branding")%>
   <div class="sage-assistant__body">
-    <div class="sage-container--sm">
-      <label for="sage-assistant__search" class="visually-hidden">Search</label>
-      <input type="text" class="sage-assistant__search" id="sage-assistant__search" placeholder="Search&hellip;">
-    </div>
```
Comment (AndrwM) [inline app/views/sage/application/_assistant.html.erb:11]: Why was this snippet removed?

**Review by AndrwM — APPROVED** [review]
Great! +1

File: app/views/sage/application/_assistant.html.erb [hunk 2]
```diff
@@ -5,10 +5,6 @@
   </button>
   <%= image_tag("sage/sage.svg", class: "sage-assistant__branding")%>
   <div class="sage-assistant__body">
-    <div class="sage-container--sm">
-      <label for="sage-assistant__search" class="visually-hidden">Search</label>
```
Comment (pixelflips) [inline app/views/sage/application/_assistant.html.erb:9]: Search was removed?

**Review by pixelflips — APPROVED** [review]
👍 Nice work!

**Review by philschanely — APPROVED** [review]

File: app/views/sage/application/_assistant.html.erb (hunk 1 above)
Comment (cmcfadzean) [reply app/views/sage/application/_assistant.html.erb:11]: The search was never hooked up its more of an in-app pattern so if we want to add search in later on we can do that as a full ticket but does not need to be cluttering the UI if it does not work currently.

File: app/views/sage/application/_assistant.html.erb (hunk 2 above)
Comment (cmcfadzean) [reply app/views/sage/application/_assistant.html.erb:9]: The search was never hooked up its more of an in-app pattern so if we want to add search in later on we can do that as a full ticket but does not need to be cluttering the UI if it does not work currently.

**Review by teenwolfblitzer — APPROVED** [review]
👍 :shipit: LGTM! Since Search was removed from the assistant, we should pull it out of the assistant example as well in a follow-up.

**Comment by philschanely** [comment]
> 👍 :shipit: LGTM! Since Search was removed from the assistant, we should pull it out of the assistant example as well in a follow-up.

If its not in this app could it still perhaps be a feature in other contexts?

**Comment by cmcfadzean** [comment]
Just a note on the search in the assistant. When we go to use this in the app there will be the search field but in the documentation its not used so it should not be in the search bar but should stay in the documentation of how the assistant should be used. Hope that makes sense.

**Pull request merged**